Applications/InSituLib/Adaptor.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "Adaptor.h"
+
+#include <vtkCPDataDescription.h>
+#include <vtkCPInputDataDescription.h>
+#include <vtkCPProcessor.h>
+#include <vtkCPPythonScriptPipeline.h>
+#include <vtkNew.h>
+#include <vtkUnstructuredGrid.h>
+
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Vtk/VtkMappedMeshSource.h"
+
+namespace InSituLib
+{
+vtkCPProcessor* Processor = NULL;
```

Comment:
For discussion (of implementation) :Better to switch to a smart pointer which will delete it at the end of the program. VTK must have something there, otherwise a `unique_ptr` with custom deleter is a good option.